VRDR.CLI/Program.cs

```diff
@@ -511,6 +511,26 @@ static int Main(string[] args)
                 Console.WriteLine(d.ToJSON());
                 return 0;
             }
+            else if (args.Length > 2 && args[0] == "ije2json")
+            {
+              string outputDirectory = args[1];
+              if (!Directory.Exists(outputDirectory))
+              {
+                  Console.WriteLine("Must supply a valid output directory");
+                  return (1);
+              }
+              for (int i = 2; i < args.Length; i++)
+              {
+                  string ijeRawRecord = args[i];
+                  string outputFilename = outputDirectory + ijeRawRecord.Substring(0, 12) + ".json";

```

Comment:
In the current format, I'm actually passing the entire raw IJE record as a string argument to this where the record is in the first 12 characters. Did this to prevent writing to an IJE file which could take longer and be redundant. Would it make more sense to first write to IJE record then convert it to JSON by passing the file name to ije2json rather than full IJE records?